package.json

```diff
@@ -110,6 +110,7 @@
     "redux-thunk": "^2.1.0",
     "sass-loader": "^3.1.2",
     "style-loader": "^0.13.0",
+    "term.js": "rancher/term.js#583581949471d833849db71cc9b1d3b74d2d8474",
```

Comment:
Even this rancher fork is broken and we should just fork it to comment out the lines where it tries to import fs.